pallets/polkadex-ido/Cargo.toml

```diff
@@ -0,0 +1,66 @@
+[package]
+name = "polkadex-ido"
+version = "0.1.0"
+authors = ["Polkadex OU"]
+edition = "2021"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { version = "3.0.0", package = "parity-scale-codec", default-features = false, features = ["derive"] }
+hex = { version = "2.1.0", package = "rustc-hex", default-features = false }
+serde = { version = "1.0.127", optional = true }
+scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
+frame-support = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+frame-system = {  git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+sp-io = {   git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19",default-features = false }
+sp-core = {   git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+sp-std = {   git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+frame-benchmarking = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", optional = true }
+orml-tokens = {  git = "https://github.com/Polkadex-Substrate/open-runtime-module-library.git", default-features = false,  branch="polkadot-v0.9.19"}
```

Comment:
We are not using orml dependencies anymore, please remove them

Reply:
Also, remove any unused dependencies.

Reply:
https://crates.io/crates/cargo-udeps may be useful

Reply:
Thanks for suggesting @jangorecki . All dependencies are being used checked by `udeps` 
```
info: Loading save analysis from "/home/felix/pdex-code/fx-ido-refactor/Polkadex/target/debug/deps/save-analysis/libpallet_polkadex_ido_primitives-811fdb0155bdd19c.json"
info: Loading save analysis from "/home/felix/pdex-code/fx-ido-refactor/Polkadex/target/debug/deps/save-analysis/polkadex_ido-0596a546f6429521.json"
info: Loading save analysis from "/home/felix/pdex-code/fx-ido-refactor/Polkadex/target/debug/deps/save-analysis/libpolkadex_ido-d85e11832f3d92e7.json"
All deps seem to have been used.
```